### ISSUE TITLE
a sort of navigation system for mechanical builds (missiles and stuff) / more detectors + a grabbing device

### ISSUE THREAD

**Issue by MrKitkat1** [studio-minus/ppg-bugs-and-requests#1078]
The feature im thinking is like a radar, im thinking that there can be a radar, and a detection device, where you can link a radar to a detection device like how the radio works with the channels, and we can set priorities so that the radar will know what to go for first, this can be used for like missiles and such that need to go in a straight line, or you can use them like nodes, where you can "program" a plane to go in a loop, back and forth, and the radar could have modes like directional or distance where directional will get direction and output a blue signal if the target is to the left and a red if its to the right and the distance one can detect the distance to it, so that you can make a rocket go to a target in the air and when within 5 m from the target, detonate, regardless of if its facing the target or not, and maybe a gyroscope that outputs data, where it will output different signals when tilted to the left or to the right. 

uses could be something like a missile that is automatically controlled, where you can use the radar tool to detect where to turn the missile, and the distance mode to detect when to detonate, or many other uses that im sure the community will love, because i sure as hell would love to see a missile that can detect a target, lock onto said target, and detonate near the target just imagine the fun of automatically killing people with your messed up contraption.

also with the radar, add data wires that can transfer data from other blocks and a global locator that will map out everything in the map, find a route for the contraption or whatever it is, and transfer the data to the radar which will lock onto any object i want without having to strap a GPS onto someone, also try adding FOV to detectors, where they can detect life around them and detonate then, or just use the radar, whatever, just point is add a navigation system for contraptions to locate, lock on, and travel to the target, also with this maybe instead of having to node everything yourself. maybe implement nodes in maps, like a pathfinding system uses, that will tell a missile to turn automatically to get to the target, so that if this does get implemented, a missile wont ram into a wall but instead will go around the wall.

also if you can add one of thoose grabbing fields (forgot the name, is it displacement field i think?) but make it change the grabbed stuff to stay in place rigidly like you pinned whatever is in there with steel bars, and make it also change them to have zero gravity, or just add a grabber thing that will grab objects with a hand, like the arms from portal 2

**Comment by DindinYT37** [comment]
I like how you wrote half a book but didn't follow the template

**Comment by ppgplayer** [comment]
> I like how you wrote half a book but didn't follow the template

i like how you read half a book but the only part you were focused on was a template

**Comment by DindinYT37** [comment]
> > I like how you wrote half a book but didn't follow the template
> 
> i like how you read half a book but the only part you were focused on was a template

I didn't read it, which is why I gave no opinion, smartass.

**Comment by ppgplayer** [comment]
> > > I like how you wrote half a book but didn't follow the template
> > 
> > 
> > i like how you read half a book but the only part you were focused on was a template
> 
> I didn't read it, which is why I gave no opinion, smartass.

sounds like somebody doesn't read suggestions, but looks for the template instead.

**Comment by DindinYT37** [comment]
> > > > I like how you wrote half a book but didn't follow the template
> > > 
> > > 
> > > i like how you read half a book but the only part you were focused on was a template
> > 
> > 
> > I didn't read it, which is why I gave no opinion, smartass.
> 
> sounds like somebody doesn't read suggestions, but looks for the template instead.

It was 1 AM and I was on the phone, I was about to go to sleep so I didn't bother reading the whole book. I did want to read it later, but I forgot about it, so thanks for the reminder.

**Comment by ppgplayer** [comment]
> > > > > I like how you wrote half a book but didn't follow the template
> > > > 
> > > > 
> > > > i like how you read half a book but the only part you were focused on was a template
> > > 
> > > 
> > > I didn't read it, which is why I gave no opinion, smartass.
> > 
> > 
> > sounds like somebody doesn't read suggestions, but looks for the template instead.
> 
> It was 1 AM and I was on the phone, I was about to go to sleep so I didn't bother reading the whole book. I did want to read it later, but I forgot about it, so thanks for the reminder.

you wanna talk about 1036?

**Comment by DindinYT37** [comment]
> > > > > > I like how you wrote half a book but didn't follow the template
> > > > > 
> > > > > 
> > > > > i like how you read half a book but the only part you were focused on was a template
> > > > 
> > > > 
> > > > I didn't read it, which is why I gave no opinion, smartass.
> > > 
> > > 
> > > sounds like somebody doesn't read suggestions, but looks for the template instead.
> > 
> > 
> > It was 1 AM and I was on the phone, I was about to go to sleep so I didn't bother reading the whole book. I did want to read it later, but I forgot about it, so thanks for the reminder.
> 
> you wanna talk about 1036?

No, because it would be off-topic. Though I can tell you that I found the issue which basically addressed the same issue, and it is #968.

**Comment by ppgplayer** [comment]
> > > > > > > I like how you wrote half a book but didn't follow the template
> > > > > > 
> > > > > > 
> > > > > > i like how you read half a book but the only part you were focused on was a template
> > > > > 
> > > > > 
> > > > > I didn't read it, which is why I gave no opinion, smartass.
> > > > 
> > > > 
> > > > sounds like somebody doesn't read suggestions, but looks for the template instead.
> > > 
> > > 
> > > It was 1 AM and I was on the phone, I was about to go to sleep so I didn't bother reading the whole book. I did want to read it later, but I forgot about it, so thanks for the reminder.
> > 
> > 
> > you wanna talk about 1036?
> 
> No, because it would be off-topic. Though I can tell you that I found the issue which basically addressed the same issue, and it is #968.

No memes in on topic!

**Comment by ahlad69** [comment]
> > > > > I like how you wrote half a book but didn't follow the template
> > > > 
> > > > 
> > > > i like how you read half a book but the only part you were focused on was a template
> > > 
> > > 
> > > I didn't read it, which is why I gave no opinion, smartass.
> > 
> > 
> > sounds like somebody doesn't read suggestions, but looks for the template instead.
> 
> It was 1 AM and I was on the phone, I was about to go to sleep so I didn't bother reading the whole book. I did want to read it later, but I forgot about it, so thanks for the reminder.

this dude deadass follows a github for some random ass game until he sleeps wth

**Comment by DindinYT37** [comment]
> > > > > > I like how you wrote half a book but didn't follow the template
> > > > > 
> > > > > 
> > > > > i like how you read half a book but the only part you were focused on was a template
> > > > 
> > > > 
> > > > I didn't read it, which is why I gave no opinion, smartass.
> > > 
> > > 
> > > sounds like somebody doesn't read suggestions, but looks for the template instead.
> > 
> > It was 1 AM and I was on the phone, I was about to go to sleep so I didn't bother reading the whole book. I did want to read it later, but I forgot about it, so thanks for the reminder.
> 
> this dude deadass follows a github for some random ass game until he sleeps wth 

Called seeing a notification.

**Comment by mestiez** [comment]
Seriously this is you guy's last warning. Don't make off-topic comments, don't reply to off-topic comments. Don't acknowledge them. Only talk about the relevant issue. Take your annoying little conversations to Reddit or Discord or whatever, not here. This is not a social board.
